src/main/java/persistence/sql/dml/QueryBuilder.java

```diff
@@ -0,0 +1,94 @@
+package persistence.sql.dml;
+
+import jakarta.persistence.*;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.stream.Collectors;
+
+import static persistence.sql.dml.parser.ValueParser.insertValuesClauseParse;
+import static persistence.sql.dml.parser.ValueParser.valueParse;
+
+public class QueryBuilder {
+
+    public String createInsertQuery(Object object) {
+        String tableName = createTableName(object.getClass());
+        String columns = columnsClause(object.getClass());
+        String values = valueClause(object);
```

Comment:
```suggestion
        String tableName = createTableName(object.getClass());
        String columns = columnsClause(object.getClass());
        String values = valueClause(object);
```
메소드로 추출해주셔서 코드의 가독성과 책임역할이 잘분리되었네요 👏

Reply:
좀 더 책임을 분리하기 위해 객체화를 고민해보면 어떨까요?
아래 코드는 느낌정도로만 참고해주세요~ ㅎㅎ
```suggestion
EntityTable table = EntityTable.of(object.getClass());
        EntityColumns columns = EntityColumns.of(object.getClass());
        
        return String.format("insert into %s (%s) values (%s)", table.name(), columns.names(), columns.values());
```

Reply:
말씀하신대로 분리해보았습니다.
한번 검토 부탁 드릴게요! 
재밌네요 ㅎ